docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-gitlab-self-hosted-managed-project.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Connect Spryker Code Upgrader to a GitLab self-hosted managed project
+description: Learn how to connect Spryker Code Upgrader to a GitLab self-hosted managed project
+template: howto-guide-template
+redirect_from:
+  - /docs/paas-plus/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-getlab-self-hosted-managed-project.html
+---
+
+## Connect Spryker Code Upgrader using GitLab CE/EE access token
+
+To connect the Spryker Code Upgrader manually using a Gitlab CE/EE access token, take the following steps.
+
+## Prerequisites
+
+[Create a GitLab access token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token)
+
+Your GitLab access token should have the following repository permissions:
+
+* **api** for Spryker CI: grants complete read and write access to the scoped project API, including the Package Registry
+
+* **write_repository** for Spryker Upgrader Service: grants read and write access to the repository to enable the Upgrader to analyze the project and create PRs.
```

Comment:
What's the difference between complete and regular access?

Reply:
I took this info from official GitLab description:

<img width="859" alt="Screenshot 2023-04-19 at 11 08 01" src="https://user-images.githubusercontent.com/111562890/233026676-4fee022b-0126-4b4c-ad2c-e240152adce0.png">